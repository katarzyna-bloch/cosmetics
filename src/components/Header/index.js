import styled from 'styled-components'

const Wrapper = styled.header`
  top: 0;
  left: 0;
  position: sticky;
  background-color: palevioletred;
  padding: 20px;
`
const Title = styled.h1`
  font-family: Arial, Helvetica, sans-serif;
`
const Header = () => (     
  <Wrapper>
    <Title>Blog o kosmetykach</Title>
  </Wrapper>
)

export default Header
