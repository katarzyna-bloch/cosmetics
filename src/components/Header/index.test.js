import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '.';

it('should render correctly', () => {
  let wrapped = shallow(<Header/>);
  expect(toJson(wrapped)).toMatchSnapshot();
});