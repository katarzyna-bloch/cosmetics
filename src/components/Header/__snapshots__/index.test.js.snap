// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<styled.h1>
  Blog o kosmetykach
</styled.h1>
`;
